Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.cpp

```diff
@@ -104,17 +105,14 @@ int CLuaCameraDefs::GetCamera(lua_State* luaVM)
     return 1;
 }
 
-int CLuaCameraDefs::GetCameraViewMode(lua_State* luaVM)
+std::tuple<int, int> CLuaCameraDefs::GetCameraViewMode()
 {
-    unsigned short ucMode;
-    if (CStaticFunctionDefinitions::GetCameraViewMode(ucMode))
-    {
-        lua_pushnumber(luaVM, ucMode);
-        return 1;
-    }
+    CClientCamera* pCamera = g_pClientGame->GetManager()->GetCamera();
 
-    lua_pushboolean(luaVM, false);
-    return 1;
+    unsigned short ucVehicleMode = (unsigned short)pCamera->GetCameraVehicleViewMode();
+    unsigned short ucPedMode = (unsigned short)pCamera->GetCameraPedViewMode();
+
+    return std::make_tuple(ucVehicleMode, ucPedMode);
```

Comment:
I think you can return `{ucVehicleMode, ucPedMode}`

Reply:
It will broke old behavior.

Reply:
It does the same. It constructs a tuple.
I didn't mean to return a table in Lua.
It's called an initializer list.

Reply:
Good stuff, thx.